.github/workflows/pr_ios.yml

```diff
@@ -25,25 +25,38 @@ jobs:
           node-version: '18'
           cache: 'npm'
 
+      - name: Cache CocoaPods
+        uses: actions/cache@v4
+        with:
+          path: e2e/ios/Pods
+          key: pods-${{ runner.os }}-${{ hashFiles('Podfile.lock') }}
+          restore-keys: |
+            pods-${{ runner.os }}-
+
+      - name: Run patch scripts
+        working-directory: e2e
+        run: bash ./scripts/patch_ios_podspec.sh
+
       - name: Run npm install
         run: |
           npm i -g corepack
           npm ci
-          cd example
+          cd e2e
           npm ci
 
-      - name: Run pod install
-        working-directory: example/ios
+      - name: Run pod install & update dependencies
+        working-directory: e2e/ios
         run: |
           pod install
+          pod update
```

Comment:
Is deze nodig? 
Een pod install zorgt ervoor dat de pods die in de repo onder de podfile.lock zitten worden geinstalleerd. Door een pod update te doen ga je de locks negeren en hogere pod versies installeren. De uiteindelijk build is dan niet tegen dezelfde pods als in de repo. 
Ik zou verwachten dat als er een pod update moet gebeuren, die dan door ons gedaan wordt en gecommit wordt in de repo. Deze e2e moet gewoon oppikken wat daarin beslist werd?

Reply:
Hoe zorgen we er dan best voor dat we altijd tegen de laatste versie van alle native dependencies builden?